client/src/utils/getFileMode.js

```diff
@@ -0,0 +1,40 @@
+export const getFileMode = ({ selectedFile }) => {
+  const extension = selectedFile.split(".").reverse()[0];
```

Comment:
Just feels like reverse is unnecessary. And better to add a condition if split generates length greater than 2. ex. .env or some built file from go.

Reply:
Thanks for the suggestion, although the reverse function shouldn't be an issue here, as the split fileName would have been a smaller number. Still, I have refactored it.

Reply:
For the env and any other one-worded extensions, it should not cause any issues as I have already added a default case.